core/templates/dev/head/pages/exploration_editor/ParamMetadataObjectFactory.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating new frontend instances of ParamMetadata
+ * domain objects.
+ */
+
+oppia.factory('ParamMetadataObjectFactory', [function() {
+  var ParamMetadata = function(action, paramName, source, sourceInd) {
+    this.action = action;
+    this.paramName = paramName;
+    this.source = source;
+    this.sourceInd = sourceInd;
+  };
+
+  ParamMetadata.createFromBackendDict = function(paramMetadataDict) {
+    return new ParamMetadata(paramMetadataDict.action,
```

Comment:
In general, when contents of parens span multiple lines, break after '('.

Reply:
Done!